live-examples/js-examples/string/string-split.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = "The quick brown fox jumped over the lazy dog.";
+
+words = str.split(" ");
+console.log(words[3]); // fox
```

Comment:
Please use the `expected output:` format (https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#indicating-consolelog-output).